CRM/Xcm/CustomData.php

```diff
@@ -15,769 +15,828 @@
 +--------------------------------------------------------*/
 
 class CRM_Xcm_CustomData {
-    const CUSTOM_DATA_HELPER_VERSION   = 0.7;
-    const CUSTOM_DATA_HELPER_LOG_LEVEL = 0;
-    const CUSTOM_DATA_HELPER_LOG_DEBUG = 1;
-    const CUSTOM_DATA_HELPER_LOG_INFO  = 3;
-    const CUSTOM_DATA_HELPER_LOG_ERROR = 5;
-
-    /** caches custom field data, indexed by group name */
-    protected static $custom_group2name       = NULL;
-    protected static $custom_group2table_name = NULL;
-    protected static $custom_group_cache      = array();
-    protected static $custom_group_spec_cache = array();
-    protected static $custom_field_cache      = array();
-
-    protected $ts_domain = NULL;
-    protected $version   = self::CUSTOM_DATA_HELPER_VERSION;
-
-    public function __construct($ts_domain) {
-        $this->ts_domain = $ts_domain;
-    }
-
-    /**
-     * Log a message if the log level is high enough
-     */
-    protected function log($level, $message) {
-        if ($level >= self::CUSTOM_DATA_HELPER_LOG_LEVEL) {
-            CRM_Core_Error::debug_log_message("CustomDataHelper {$this->version} ({$this->ts_domain}): {$message}");
-        }
+  const CUSTOM_DATA_HELPER_VERSION   = '0.9-dev';
```

Comment:
@bjendres Let's please use tagged non dev [versions](https://github.com/systopia/Custom-Data-Helper/tags) here

Reply:
> Let's please use tagged non dev [versions](https://github.com/systopia/Custom-Data-Helper/tags) here

Good catch. Updated to 0.10, please test again @jofranz. If it works we can merge and release